src/components/victory-group/victory-group.js

```diff
@@ -402,7 +408,9 @@ export default class VictoryGroup extends React.Component {
         labels,
         theme: child.props.theme || props.theme,
         labelComponent: props.labelComponent || child.props.labelComponent,
-        domainPadding: child.props.domainPadding || props.domainPadding,
+        domainPadding: child.props.domainPadding
```

Comment:
extract into method, pad Y for horizontal charts. Make sure domainPadding is working for horizontal charts